pkg/sql/catalog/descpb/structured.proto

```diff
@@ -1180,7 +1180,10 @@ message TableDescriptor {
   // this table, in which case the global setting is used.
   optional bool forecast_stats = 52 [(gogoproto.nullable) = true, (gogoproto.customname) = "ForecastStats"];
 
-  // Next ID: 54
+  // ImportStartWallTime is the start wall time of an in-progress import, if it exists.
```

Comment:
Should only ever be non-zero if table is offline, right?

Reply:
Also, Schema might have feelings about the name of this field, e.g. should it be a generic offline-after or something like that (once upon a time someone wanted to automatically move implicit txns back to the last online time for a table to hide the fact it was offline)

Reply:
Yes, this field is nonzero if (not IFF though) the table is offline during an import. clarified that in the proto docstring.

I'd rather keep the field specific to imports, since in the IMPORT INTO non-empty table case, the table goes offline _before_ the import wall time and this descriptor field get set and written to disk.